Makefile

```diff
@@ -71,15 +78,20 @@ check: python-env
 # Corrects spelling errors
 .PHONY: misspell
 misspell:
-	go get github.com/client9/misspell
-	${FIND} -name '*'  -exec misspell -w {} \;
+	go get $(MISSPELL_REPO)
```

Comment:
The nicest thing would be if we could somehow load a yaml file. So we would have things like `beat_name` or `beat_version` also available in other tools. But I don't know if there is a way in Makefiles to load env variables from a file.